charts/fluent-bit/templates/_pod.tpl

```diff
@@ -54,14 +54,17 @@ containers:
     envFrom:
       {{- toYaml .Values.envFrom | nindent 6 }}
   {{- end }}
-  {{- if .Values.args }}
-    args:
-    {{- toYaml .Values.args | nindent 6 }}
-  {{- end}}
-  {{- if .Values.command }}
+  {{- with .Values.command }}
     command:
-    {{- toYaml .Values.command | nindent 6 }}
+      {{- toYaml . | nindent 6 }}
   {{- end }}
+  {{- if or .Values.args .Values.hotReload.enabled }}
+    args:
+      {{- toYaml .Values.args | nindent 6 }}
+    {{- if .Values.hotReload.enabled }}
```

Comment:
We also need the webserver enabled for this, I'm not sure that is configurable (probably should be) as it is part of the explicit config - and the port open for it.

Reply:
Fluent Bit shouldn't be running in Kubernetes without the webserver enabled for probes (and metrics), if it is running the sidecar will be able to access it.

Reply:
Well, you can disable those too if you really want :) 
People have also disabled the webserver and then got failing probes because it is not always clear.
But agreed.